### PR TITLE
Project.toml

### PR DESCRIPTION
**Pull request by mileslucas**
Hello,

I've taken the current Master branch and added a `Project.toml`, deleting the old REQUIRE files in base and test. I've matched the `[compat]` section with the versions from REQUIRE. 

In addition I've added julia 1.1 to the CI matrices since nightly now points to 1.2 

The current version is `1.0.1` based on what I saw in NEWS.md

I wasn't able to get the tests to pass, but this seems like an issue not related directly to project.toml but perhaps compatibility issues. If it's the latter we just have to pin the correct versions of dependencies.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+name = "GaussianProcesses"
+uuid = "482d963d-215c-4c64-a1e6-68ef95496ddc"
+license = "MIT"
+version = "1.0.1"
```
Comment (jbrea) [inline Project.toml]: Should be "1.0.0".

**Comment by jbrea** [comment]
Hi @mileslucas 

Thanks for this PR.
It looks all fine to me except that the version should be 1.0.0.
NEWS.md was updated in preparation of the new release.
We will fix the failing tests outside of this PR.

**Comment by mileslucas** [comment]
Updated the version number to be `1.0.0`

**Comment by fairbrot** [comment]
I created a Project.toml in October without seeing this pull request so unfortunately it is now redundant. Apologies!

**Pull request closed without merging**